src/measures.jl

```diff
@@ -83,6 +83,23 @@ function measure(t::Tetrahedron)
   abs((A - D) ⋅ ((B - D) × (C - D))) / 6
 end
 
+"Compute the measure (i.e. hyper-volume) for a simplex in any dimension using the Cayley-Menger Determinant."
+function measure(splx::Simplex{K,Dim,T,N}) where {K, Dim, T<:Real, N}
+  distancebuf = Matrix{T}(undef, N+1, N+1)
+  measure!(distancebuf, splx)
+end
+
+"In-place verion of [`measure`](@ref)."
+function measure!(distancebuf::AbstractMatrix{T}, splx::Simplex{K,Dim,T,N}) where {K, Dim, T<:Real, N}
+  size(distancebuf) == (N+1, N+1) || throw(ArgumentError("Buffer dimensions must be (N+1) times (N+1)."))
```

Comment:
In the Julia documentation, the recommendation is that error messages start with a lowercase letter, unless the argument or type name is written with a capital letter:
![image](https://github.com/JuliaGeometry/Meshes.jl/assets/73039601/731cd4ff-f88a-4e2c-a528-ccf160641229)